### PR TITLE
Add easy-blog app

### PR DESCRIPTION
**Pull request by akirk**
This adds support for the Easy Blog app. I have only tested the basic app so far which is quite primitive:

Screenshot 2021-02-24 at 16 16 51

Hence I was unable to use images but I believe they work well because I was able to re-use much of site-parser's code.

### CODE REVIEW

**Comment by akirk** [comment]
@pento this is now ready for review. I was able to re-use quite a bit of code from site-parser, and Gutenberg's `pasteHandler` does the magic of converting the HTML to blocks. I basically worked on this solely in node by using a HAR file and fixtures with the integration test. This went very smooth.

File: source/services/wix/extractors/easy-blog-app/index.js [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+		data.forEach( ( post ) => {
+			const postTags = post.tags.map( ( postTag ) => ( {
+				type: 'tag',
+				slug: postTag,
+				name: postTag,
+			} ) );
+
+			wxr.addPost( {
+				// guid: post.id, // No unique ids.
+				// author: postAuthor.slug, // No support for authors.
```
Comment (pento) [inline source/services/wix/extractors/easy-blog-app/index.js]: These look like leftover comments? They can probably be removed.

File: source/services/wix/extractors/easy-blog-app/index.js [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+				date: post.date,
+				title: post.title,
+				content: pasteHandler( { HTML: post.content } )
```
Comment (pento) [inline source/services/wix/extractors/easy-blog-app/index.js]: This is probably a safe use of `pasteHandler()`, as the content is fairly simple. It's probably worth setting `mode: 'BLOCKS'`, though, to ensure it isn't accidentally treated as inline content.

**Review by pento — APPROVED** [review]
Looks good, I'm glad you were able to reuse a bunch of code from site-parser here.

A couple of minor comments, but this is otherwise good to merge.

File: package.json [hunk 3]
```diff
@@ -23,7 +23,9 @@
 		"@wordpress/format-library": "1.26.5",
 		"@wordpress/rich-text": "3.24.5",
 		"@wordpress/wxr": "file:packages/wxr",
+		"cheerio": "1.0.0-rc.5",
 		"file-saver": "2.0.5",
+		"moment": "2.29.1",
```
Comment (yoavf) [inline package.json:28]: A late note - we should probably stay away from momentjs, as it's super bloated. Some good alternatives listed [here](https://github.com/you-dont-need/You-Dont-Need-Momentjs).

**Pull request merged**